about.html

```diff
@@ -134,124 +134,241 @@
       <div class="text-content">
         <h1>About Mother Tongues Network</h1>
 
-				<p>
-					Mother Tongues Network (MTN) was founded by a passionate team of
-					individuals committed to preserving and celebrating Indigenous
-					languages and cultures. Our co-founders, Lianna Rice, Miranda
-					Saddleback, and Kyla Monture, bring together expertise in
-					operations, community engagement, social work, communications, and
-					marketing to drive MTN's mission forward. Our team of web developers
-					brings a diverse range of expertise, with members comprising of
-					Indigenous and non-Indigenous professionals, some of whom have
-					extensive backgrounds in technology spanning over a decade, while
-					others bring experience in childcare, healthcare, and animation to
-					the table.
-				</p>
-			</div>
-		</section>
+        <p>
+          Mother Tongues Network (MTN) was founded by a passionate team of
+          individuals committed to preserving and celebrating Indigenous
+          languages and cultures. Our co-founders, Lianna Rice, Miranda
+          Saddleback, and Kyla Monture, bring together expertise in
+          operations, community engagement, social work, communications, and
+          marketing to drive MTN's mission forward. Our team of web developers
+          brings a diverse range of expertise, with members comprising of
+          Indigenous and non-Indigenous professionals, some of whom have
+          extensive backgrounds in technology spanning over a decade, while
+          others bring experience in childcare, healthcare, and animation to
+          the table.
+        </p>
+      </div>
+    </section>
+
+    <section class="commitment right-fade wrapper">
+      <div class="text-content">
+        <h2>Commitment to Authenticity</h2>
+        <p>
+          As part of our commitment to authenticity and cultural relevance, we
+          collaborate closely with Indigenous language experts, educators, and
+          community members to curate high-quality language resources for our
+          Indigenous Language Learning App. Together, we strive to create a
+          supportive and inclusive community where language learners can
+          connect, share experiences, and learn from one another.
+        </p>
+      </div>
+      <div class="commitment-img">
+        <img
+          src="./assets/mtn-content-pics/trc-jerry-saddleback.png"
+          alt="" />
+      </div>
+    </section>
+
+
+
+    <div class="team-container">
+      <div class="team-title">
+        <h2>Meet the Team</h2>
+        <p>Click photo to view bio</p>
+      </div>
+      <ul class="team-members">
+        <li class="person" data-modal="lianna-modal">
+          <img src="./assets/MTN Team Photos/Lianna-personal-pic.jpeg" alt="Lianna Rice" id="lianna-img">
```

Comment:
Added alts. 

I don't think we need the ids anymore as the JavaScript is now using the `person` class and the `data-modal` attribute.